WordPress/Classes/ViewRelated/Stats/Subscribers/SubscribersChartMarker.swift

```diff
@@ -0,0 +1,35 @@
+import DGCharts
+
+class SubscribersChartMarker: StatsChartMarker {
+    let date: Date
+
+    init(dotColor: UIColor, name: String, date: Date) {
+        self.date = date
+        super.init(dotColor: dotColor, name: name)
+    }
+
+    public required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func text(for entry: ChartDataEntry) -> NSMutableAttributedString? {
+
+        let subscriberCountRowAttributes: [NSAttributedString.Key: Any] = [.font: UIFont.preferredFont(forTextStyle: .headline),
+                                                                           .paragraphStyle: paragraphStyle,
+                                                                           .foregroundColor: UIColor.white]
+
+        let subscriberLabelRowAttributes: [NSAttributedString.Key: Any] = [.font: UIFont.preferredFont(forTextStyle: .footnote),
+                                                                           .paragraphStyle: paragraphStyle,
+                                                                           .foregroundColor: UIColor.white]
+
+        let subscriberDateRowAttributes: [NSAttributedString.Key: Any] = [.font: UIFont.preferredFont(forTextStyle: .footnote),
+                                                                          .paragraphStyle: paragraphStyle,
+                                                                          .foregroundColor: UIColor.white.withAlphaComponent(0.8)]
+
+        let attrString = NSMutableAttributedString()
```

Comment:
We could make the line spacing a bit smaller to align more closely with designs:

![image](https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/7e6468f2-1aad-4dd4-945b-c19b424a4fe1)